examples/nodejs/cache/leaderboard.ts

```diff
@@ -0,0 +1,88 @@
+import {LeaderboardClient, LeaderboardConfigurations, CredentialProvider, SortedSetOrder} from '@gomomento/sdk';
+
+async function main() {
+  const client = new LeaderboardClient({
+    configuration: LeaderboardConfigurations.Laptop.v1(),
+    credentialProvider: CredentialProvider.fromEnvironmentVariable({
+      environmentVariableName: 'MOMENTO_API_KEY',
+    }),
+  });
+
+  // upsert
+  const elements: Map<bigint, number> = new Map([
+    [123n, 100],
+    [456n, 200],
+    [789n, 300],
+  ]);
+  const upsertResp = await client.leaderboardUpsert(
+    "my-cache", 
+    "my-leaderboard", 
+    elements
+  );
+  console.log("Upsert elements:", upsertResp.toString());
```

Comment:
we need to include some error handling stuff here, at least in the first few APIs if not all of them.